packages/amplify-graphiql-explorer/package.json

```diff
@@ -29,7 +29,7 @@
     "eslint-webpack-plugin": "^3.1.1",
     "file-loader": "^6.2.0",
     "fs-extra": "^10.0.0",
-    "graphiql": "^1.5.16",
+    "graphiql": "1.5.16",
```

Comment:
This change is to resolve the following build error

![image](https://github.com/aws-amplify/amplify-cli/assets/11983489/8074d399-c56a-4916-8f51-9b08e601d0cc)

With `^`, `graphiql` would use `1.11.x`

Reply:
This change is not necessary anymore since `yarn.lock` is fixed